### ISSUE TITLE
Suggestion on creating a road map for graduates 

### ISSUE THREAD

**Issue by Heathercoraje** [foundersandcoders/master-reference#743]
As a new graduate, I have found myself quite lost collecting and deciding what to learn to where direction to take since the course is completely over. Though it is true that there are plenty of resource one can teach oneself with but I thought it would come handy if there is a road-map for graduates to get help from.
I would love to hear some thoughts.

**Comment by veredrec** [comment]
I agree @Heathercoraje, I think some kind of a guide might be helpful. 
Since there are so many different paths that we as graduates can take, I was thinking maybe we make a collection of FAC graduates' paths and stories? Maybe each graduate (from all campuses) can write what steps they took after graduating and what they are doing today. 
I think it can be a useful resource for us as an organization, and it can also be used as a tool for students/graduates to know who they can ask about a certain direction or experience. :woman_student: :man_student:

**Comment by m4v15** [comment]
I agree, I think it can be hard for a lot of people when we ask them early on in the course what they want to do afterwards - most of the time the answer is something that boils down to "What are my options?". I think often in the past this has been covered by the knowledge of the mentors and or alumni that are around, which is great but not totally sustainable or repeatable.  Having a resource of past alumni's career progression would be amazing for this.

@yvonne-liu @rebeccaradding @sofer do we have any info/existing docs on past graduates experiences? I would be happy to get involved in constructing a google doc/survey we could send out to alumni (maybe in the newsletter or something?)

**Comment by Heathercoraje** [comment]
I agree with @m4v15 and @veredrec as in collecting docs of progress one has made to land a first job would be a great idea. I also think it'd be nice to create a mini course book-like materials of must-knows when applying to first job along with docs of past graduates experience.

**Comment by zurda** [comment]
I came across this useful link recently: [developer-roadmap](https://github.com/kamranahmedse/developer-roadmap), and I agree with the above discussion. Creating a list of free/valuable resources for graduates would be extremely useful since we do have time for more learning during the job search
